### PR TITLE
New package: andem81.Windroy version 0.5.5

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118821)

### CODE REVIEW

**Comment by stephengillie** [comment]
Please add a PackageUrl.

**Review by Trenly — CHANGES_REQUESTED** [review]
AppsAndFeaturesEntries are required as the DisplayVersion does not match PackageVersion
```
--> Installing the Manifest 4.0.3

Found Windroy [andem81.Windroy] Version 4.0.3
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://cytranet.dl.sourceforge.net/project/windroy/4.0.3/windroy_20140113.exe
  ██████████████████████████████  79.3 MB / 79.3 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName           DisplayVersion Publisher     ProductCode
-----------           -------------- ---------     -----------
Windroy version 0.5.5 0.5.5          Windroy, Inc. {7E07052F-A4CE-4932-B066-B9203888439F}_is1
```

**Comment by upintheairsheep** [comment]
> AppsAndFeaturesEntries are required as the DisplayVersion does not match PackageVersion
> 
> ```
> --> Installing the Manifest 4.0.3
> 
> Found Windroy [andem81.Windroy] Version 4.0.3
> This application is licensed to you by its owner.
> Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
> Downloading https://cytranet.dl.sourceforge.net/project/windroy/4.0.3/windroy_20140113.exe
>   ██████████████████████████████  79.3 MB / 79.3 MB
> Successfully verified installer hash
> Starting package install...
> Successfully installed
> 
> --> Refreshing environment variables
> 
> --> Comparing ARP Entries
> 
> DisplayName           DisplayVersion Publisher     ProductCode
> -----------           -------------- ---------     -----------
> Windroy version 0.5.5 0.5.5          Windroy, Inc. {7E07052F-A4CE-4932-B066-B9203888439F}_is1
> ```

Change version to fit

**Comment by upintheairsheep** [comment]
Keepalive

**Comment by upintheairsheep** [comment]
Can someone help rename the folder?

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\a\andem81\Windroy\4.0.3', expected 
'manifests\a\andem81\Windroy\0.5.5'

File: manifests/a/andem81/Windroy/0.5.5/andem81.Windroy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: andem81.Windroy
+PackageVersion: 0.5.5
+PackageLocale: en-US
+Publisher: Windroy, Inc.
+PackageName: Windroy
```
Comment (stephengillie) [inline manifests/a/andem81/Windroy/0.5.5/andem81.Windroy.locale.en-US.yaml:8]:
```suggestion
PackageName: Windroy
PackageUrl: https://cytranet.dl.sourceforge.net/project/windroy
```

**Pull request closed without merging**